tasks/newrelic.yml

```diff
@@ -1,10 +1,12 @@
 ---
-- name: optionally add newrelic repo
-  apt_repository: repo="deb http://apt.newrelic.com/debian/ newrelic non-free" state=present
-
 - name: add the newrelic repo key
   apt_key: url="https://download.newrelic.com/548C16BF.gpg" state=present
 
+- name: optionally add newrelic repo
```

Comment:
not clear to me what "optionally" refers to in some of these task names since I don't see a `when:`

is the `newrelic.yml` task itself an option?